tests/matlab_test_data_collectors/python_testers/test_linear_array_transducer.py

```diff
@@ -0,0 +1,101 @@
+from tests.matlab_test_data_collectors.python_testers.utils.record_reader import TestRecordReader
+from tests.matlab_test_data_collectors.python_testers.utils.check_equality import check_kwave_array_equality
+from tests.matlab_test_data_collectors.python_testers.utils.check_equality import check_kgrid_equality
+# import matplotlib.pyplot as plt
+import numpy as np
+import os
+from pathlib import Path
+
+import kwave.data
+# from kwave.kWaveSimulation import SimulationOptions
+from kwave.kgrid import kWaveGrid
+from kwave.kmedium import kWaveMedium
+from kwave.ksensor import kSensor
+from kwave.ksource import kSource
+# from kwave.kspaceFirstOrder3D import kspaceFirstOrder3DC
+# from kwave.options.simulation_execution_options import SimulationExecutionOptions
+from kwave.utils.kwave_array import kWaveArray
+# from kwave.utils.plot import voxel_plot
+from kwave.utils.signals import tone_burst
+
+
+def test_linear_array_transducer():
+    test_record_path = os.path.join(Path(__file__).parent, 'collectedValues/linear_array_transducer.mat')
+    reader = TestRecordReader(test_record_path)
+
+    c0 = 1500
+    rho0 = 1000
+    source_f0 = 1e6
+    source_amp = 1e6
+    source_cycles = 5
+    source_focus = 20e-3
+    element_num = 15
+    element_width = 1e-3
+    element_length = 10e-3
+    element_pitch = 2e-3
+    translation = kwave.data.Vector([5e-3, 0, 8e-3])
+    rotation = kwave.data.Vector([0, 20, 0])
+    grid_size_x = 40e-3
+    grid_size_y = 20e-3
+    grid_size_z = 40e-3
```

Comment:
Using the `Vector` class here might be better. It will help to avoid handling each item separately.